Makefile

```diff
@@ -24,7 +26,7 @@ test-unit:
 test-functional:
 	pipenv run ./scripts/run_tests_functional.sh
 
-validate-test-schemas:
+validate-test-schemas: run-validator
```

Comment:
The naming on these commands was changed to indicate they're running independently of one another for the github workflows. Making run a prerequisite now makes this out of sync with the  workflow and other action repos.